lib/ext/ts.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+var fs = require('fs');
+
+var typescript;
+try {
+	typescript = require('typescript');
+} catch (err) {
+	console.error('AVA: `--extensions=ts` require TypeScript to be installed.');
+	throw err;
```

Comment:
Rather than logging and then rethrowing the original error construct a new error like in https://github.com/avajs/ava/blob/0f0400158ec4192fc32ccc603877d0b088302662/lib/cli.js#L99 and throw that. It'll be logged correctly.